symplyphysics/laws/optics/interference_maximum.py

```diff
@@ -3,8 +3,15 @@
                            validate_input, validate_output, dimensionless)
 
 # Description
-## If the optical distance difference is equal half-integer number of waves,
-## then at this point of the screen the interference intensity will be maximum.
+## 1) If the difference in the course of two waves is equal to an integer number of waves
+## (i.e. an even number of half-waves) Δ = mλ, where m = 0, 1, 2, ...,
+## then an interference maximum is formed at the point of superposition of these waves.
+
+## 2) An interference pattern occurs when the path difference from the slit to the
+## screen is equal to an integer number of wavelengths (0, λ, 2λ, …).
```

Comment:
I think you are wrong here. Interference pattern occurs for any coherent waves. But at maximum condition you will see light dot on the pattern.